umake/frameworks/ide.py

```diff
@@ -138,12 +133,264 @@ def is_installed(self):
         # check path and requirements
         if not super().is_installed:
             return False
-        if not isfile(join(self.install_path, "eclipse")):
+        if not isfile(join(self.install_path, "bin", self.executable)):
             logger.debug("{} binary isn't installed".format(self.name))
             return False
         return True
 
 
+class Eclipse(BaseEclipse):
+    """The Eclipse Foundation Luna distribution."""
+    download_page_url = "https://www.eclipse.org/downloads/download.php?" \
+                        "file=/technology/epp/downloads/release/luna/R/" \
+                        "eclipse-standard-luna-R-linux-gtk{arch}.tar.gz{suf}" \
+                        "&r=1"
+    executable = 'eclipse.sh'
+
+    def __init__(self, category):
+        super().__init__(name="Eclipse",
+                         description=_("Pure Eclipse Luna (4.4)"),
```

Comment:
Do not add any notion of Luna (that would be removed anyway? See my incoming general comment) or version in the .desktop file.

-> Most of tools autoupdate (I know, Eclipse doesn't really, but I hope this will come…) and we don't change the .desktop files automatically. So we prefer to keep the description functional as ("Eclipse for Java Developers"). Same comments for all other frameworks below.